docs/user/dql/functions.rst

```diff
@@ -219,7 +219,7 @@ INTEGER/LONG/FLOAT/DOUBLE -> DOUBLE
 
 Example::
 
-    opensearchsql> SELECT CBRT(8), CBRT(9.261), CBRT(-27);
+    os> SELECT CBRT(8), CBRT(9.261), CBRT(-27)
```

Comment:
Are these related to position? If not, it may be worthwhile mentioning why it's included in the PR description.

Reply:
These tests will run only when started with `os>` , otherwise they will be skipped. After Yuri mentioned that with regards to these position() related PRs, I identified  the above and got it fixed as soon as possible.